camelot/core/serializable.py

```diff
@@ -53,6 +54,10 @@ class DataclassEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, ugettext_lazy):
             return str(obj)
+        if isinstance(obj, QtGui.QKeySequence):
+            return obj.toString()
+        if isinstance(obj, QtGui.QKeySequence.StandardKey):
+            return QtGui.QKeySequence(obj).toString()
```

Comment:
De camelot unit testen falen door die toString():
```
======================================================================
ERROR: test_qt_incompatible (test.test_source.SourceQualityCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/jobs/vFinance_pipeline_timvdm/workspace/camelot/test/test_source.py", line 54, in test_qt_incompatible
    raise Exception( '%s in %s/%s'%( expr,
Exception: toString() in /var/lib/jenkins/jobs/vFinance_pipeline_timvdm/workspace/camelot/test/../camelot/core/serializable.py
```
Ik neem aan dat dit eerder voor QVariants etc. is? Kunnen we camelot/core/serializable.py toevoegen als uitzondering?

Reply:
kan het me idd nt meer perfect herinneren

dat komt ergens van de PyQt4 naar PyQt5 migratie

mogelijk idd ter verwijdering van QVariant (die expliciet was in PyQt4 en in PyQt5 direct de python versie vh object geeft)

andere mogelijkheid is dat str(x) gelijk zou zijn aan x.toString (maar daar kan ik toch nt direct documentatie van vinden)

Reply:
str(x) werkt niet, dit geeft waarde als "<PyQt6.QtGui.QKeySequence object at 0x7f95ac24aa50>" ipv de keys.